vm/src/builtins/bytearray.rs

```diff
@@ -373,9 +375,15 @@ impl PyByteArray {
         self.inner().hex(sep, bytes_per_sep, vm)
     }
 
-    #[pymethod]
-    fn fromhex(string: PyStrRef, vm: &VirtualMachine) -> PyResult<PyByteArray> {
-        Ok(PyBytesInner::fromhex(string.as_str(), vm)?.into())
+    #[pyclassmethod]
+    fn fromhex(cls: PyTypeRef, string: PyStrRef, vm: &VirtualMachine) -> PyResult {
+        let bytes = PyBytesInner::fromhex(string.as_str(), vm)?;
+        let bytes = vm.ctx.new_bytes(bytes);
+        let args = FuncArgs {
+            args: vec![bytes],
+            ..Default::default()
+        };
```

Comment:
better way to get FuncArgs from Vec<u8>?

Reply:
I think `FuncArgs::from(vec![..])` works